addons/l10n_ec/i18n/es_EC.po

```diff
@@ -16,400 +16,342 @@ msgstr ""
 "Plural-Forms: \n"
 
 #. module: l10n_ec
-#: code:addons/l10n_ec/models/l10n_ec_witholding.py:0
-#: model:ir.model.fields,field_description:l10n_ec.field_l10n_ec_witholding__witholding_move_id
-#, python-format
-msgid "Account Move"
-msgstr ""
+#: model:ir.model.fields.selection,name:l10n_ec.selection__account_tax__l10n_ec_code_taxsupport__00
+#: model:ir.model.fields.selection,name:l10n_ec.selection__account_tax_template__l10n_ec_code_taxsupport__00
+msgid "00 Special cases whose support does not apply to the above options"
+msgstr "00 Casos especiales cuyo sustento no aplica en las opciones anteriores"
 
 #. module: l10n_ec
-#: code:addons/l10n_ec/models/l10n_ec_witholding.py:0
-#: model:ir.model.fields,field_description:l10n_ec.field_l10n_ec_witholding_lines__amount
-#, python-format
-msgid "Amount"
-msgstr ""
+#: model:ir.model.fields.selection,name:l10n_ec.selection__account_tax__l10n_ec_code_taxsupport__01
+#: model:ir.model.fields.selection,name:l10n_ec.selection__account_tax_template__l10n_ec_code_taxsupport__01
+msgid ""
+"01 Tax credit for VAT declaration (services and goods other than inventories"
+" and fixed assets)"
+msgstr "01 Crédito Tributario para declaración de IVA (servicios y bienes distintos de inventarios y activos fijos)"
 
 #. module: l10n_ec
-#: code:addons/l10n_ec/models/account_move.py:0
-#: model:ir.model.fields,field_description:l10n_ec.field_account_bank_statement_line__l10n_ec_access_key
-#: model:ir.model.fields,field_description:l10n_ec.field_account_bank_statement_line__l10n_ec_auth_type
-#: model:ir.model.fields,field_description:l10n_ec.field_account_move__l10n_ec_access_key
-#: model:ir.model.fields,field_description:l10n_ec.field_account_move__l10n_ec_auth_type
-#: model:ir.model.fields,field_description:l10n_ec.field_account_payment__l10n_ec_access_key
-#: model:ir.model.fields,field_description:l10n_ec.field_account_payment__l10n_ec_auth_type
-#: model:ir.model.fields,field_description:l10n_ec.field_l10n_latam_document_type__l10n_ec_authorization
-#, python-format
-msgid "Authorization"
-msgstr ""
+#: model:ir.model.fields.selection,name:l10n_ec.selection__account_tax__l10n_ec_code_taxsupport__02
+#: model:ir.model.fields.selection,name:l10n_ec.selection__account_tax_template__l10n_ec_code_taxsupport__02
+msgid ""
+"02 Cost or Expense for IR declaration (services and goods other than "
```

Comment:
@odoostan Why do we still have these terms in here? (We should only add what we have in community)

Reply:
Removed by you.